Localization/Italian/ItalianBatteryModeLanguage.rc

```diff
@@ -0,0 +1,158 @@
+STRINGTABLE
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
+{
+0, "Italiano"
+1, "Modalità batteria"

```

Comment:
This string was meant to be the name of the program, so let's leave it untranslated :)